src/store.js

```diff
@@ -1,12 +1,13 @@
 import Vue from "vue";
 import Vuex from "vuex";
 import register from "./register/store/Register";
+import profile from "./profile/store/Profile";
```

Comment:
Rename file `./profile/store/Profile` to `./profile/store/index.js` then we can import `@/profile/store`
```suggestion
import profile from "@/profile/store";
```